### PR TITLE
Use navigator platform to display separate for Linux OS - 4848

### LINKED ISSUE #4848: Image alt-drag to pan not working in Ubuntu, but should
#### Summary
See title. Alt-dragging moves the entire browser window instead of panning the image as expected. Ctrl-alt drag does work to pan the image though.

#### Expected vs Current Behavior
See title. Solution may be to provide a different hint message utilizing OS detection via JavaScript.

#### Impact Check List

- [ ] Data loss or misrepresented data?
- [ ] Regression? Did this used to work or has it always been broken?
- [x] Is there a workaround available?
- [x] Does this impact a critical component?
- [ ] Is this just a visual bug with no functional impact?

#### Environment
* Open MCT Version: 1.8.4
* Deployment Type: VIPER/YAMCS
* OS: Linux Ubuntu
* Browser: Firefox

### PR DESCRIPTION
**Pull request by michaelrogers**
Closes #4848 and #5023

### Describe your changes:

- [x] Describe different alt text based on platform
- [x] Update tests to differentiate the help text based on platform.
- [x] Also adds some timing consistency to address #5023 

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Unit tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue?

### Reviewer Checklist

* [x] Changes appear to address issue?
* [x] Changes appear not to be breaking changes?
* [ ] Appropriate unit tests included?
* [x] Code style and in-line documentation are appropriate?
* [x] Commit messages meet standards?
* [x] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [x] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

**Review by unlikelyzero — APPROVED** [review]
This is amazing. We should sync on modifying the test to only run in specific ways on specific 'projects'

File: e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js [hunk 1]
```diff
@@ -180,6 +180,40 @@ test.describe('Example Imagery', () => {
         expect(resetBoundingBox.width).toEqual(initialBoundingBox.width);
     });
 
+    test('Image Pan/Zoom alt text is displayed (Linux)', async ({ page }) => {
```
Comment (unlikelyzero) [inline e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js:183]: test.skip(process.platform != 'darwin', 'Image Plan / Zoom alt text is displayed (Linux)', async ({ page, browserName })

**Review by unlikelyzero — CHANGES_REQUESTED** [review]
Let's try to evaluate this at the test framework level

**Review by shefalijoshi — APPROVED** [review]
Looks good to me!

File: e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js [hunk 2]
```diff
@@ -77,9 +81,11 @@ test.describe('Example Imagery', () => {
 
     test('Can use alt+drag to move around image once zoomed in', async ({ page }) => {
         const deltaYStep = 100; //equivalent to 1x zoom
+        const panHotkey = process.platform === 'linux' ? ['Control', 'Alt'] : ['Alt'];
```
Comment (michaelrogers) [inline e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js:84]: @johnhill2 - Disappointingly Playwright failed to parse a modifier sequence like `Control+Alt` so the test had to be modified to map over an array of key presses.

File: e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js [hunk 3]
```diff
@@ -179,39 +198,62 @@ test.describe('Example Imagery', () => {
...
         expect(resetBoundingBox.width).toEqual(initialBoundingBox.width);
     });
-
-    //test('Can use Mouse Wheel to zoom in and out of previous image');
-    //test('Can zoom into the latest image and the real-time/fixed-time imagery will pause');
-    //test.skip('Can zoom into a previous image from thumbstrip in real-time or fixed-time');
-    //test.skip('Clicking on the left arrow should pause the imagery and go to previous image');
-    //test.skip('If the imagery view is in pause mode, it should not be updated when new images come in');
-    //test.skip('If the imagery view is not in pause mode, it should be updated when new images come in');
+    // test('Image Pan/Zoom alt text is displayed (Linux)', async ({ page }) => {
```
Comment (unlikelyzero) [inline e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js:201]: @michaelrogers do you mind deleting these?

**Review by unlikelyzero — APPROVED** [review]

**Pull request merged**